ext/mysqli/mysqli.stub.php

```diff
@@ -22,59 +19,41 @@ final class mysqli_driver
 
 class mysqli
 {
-    /** @var int|string|null */
-    public $affected_rows;
+    public int|string|null $affected_rows;
```

Comment:
All the affected_rows properties should have an assert. It should not be possible to call it on an invalid object.